x/bank/spec/04_events.md

```diff
@@ -11,22 +11,21 @@ The bank module emits the following events:
 ### MsgSend
 
 | Type     | Attribute Key | Attribute Value    |
-| -------- | ------------- | ------------------ |
+|----------|---------------|--------------------|
 | transfer | recipient     | {recipientAddress} |
+| transfer | sender        | {senderAddress}    |
 | transfer | amount        | {amount}           |
-| message  | module        | bank               |
-| message  | action        | send               |
```

Comment:
Why `message.action` was removed at document?

Reply:
Some modules do not have the event of type `message' in its document. So, I decided to put the event into the common document, and remove it from the individual module's.